openlibrary/components/AuthorIdentifiers.vue

```diff
@@ -0,0 +1,133 @@
+<template>
+  <table>
+    <tr>
+      <th>
+        <select class="form-control" v-model="selectedIdentifier" name="name">
+          <option disabled value="">Select one</option>
+          <option v-for="idConfig in identifierConfigsByKey" :key="idConfig.name" :value="idConfig.name">
```

Comment:
```suggestion
          <option v-for="idConfig in authorConfig.identifiers" :key="idConfig.name" :value="idConfig.name">
```

Reply:
In the comment on line 61 you said we should be using `identifierConfigsByKey` directly. Is this suggestion perhaps mistyped or leftover from before?

We don't have a variable `authorConfig` right now since we are parsing the string and then just using that directly for `identifierConfigsByKey`